Update readmes (#59)
Sindri-Labs/sindri-resources: circuit_tutorials/circom/README.md

# Circom Circuit Tutorials

> In this directory, we provide a handful of Circom circuits tutorials which have been prepared to be used with Sindri.

Reviewer: ```suggestion
In this directory, we provide a handful of Circom circuits tutorials which have been prepared for use with Sindri.
```
